ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml

```diff
@@ -7,7 +7,8 @@
     container_root: '{{ container_root }}'
     configlet_dir: '{{playbook_dir}}/intended/configs'
```

Comment:
Should be updated with cherry-picking commit a788b70ebf6bef282146bbe1b0f43941b6a93066

Reply:
done